Documentation/ceph-nfs-crd.md

```diff
@@ -72,55 +82,395 @@ spec:
     priorityClassName:
 ```
 
```

Comment:
@BlaineEXE : I have a generic question, is it planned to support CRs for `Exports` in near future?

Reply:
Seb and I believe that exports are something that should be enabled by a new mode added to the Ceph-CSI driver rather than CRs.